stylesheets/pages/index.css

```diff
@@ -0,0 +1,310 @@
+@import url("../base/colors.css");
+@import url("../base/variables.css");
+
+.display-none {
+  display: none;
+}
```

Comment:
You can move this to your library's utility class and then use it across.

Reply:
Actually I wasn't using this class anymore, I forgot to remove it. Changes done, thanks